docker-hub/onboarding-faqs.md

```diff
@@ -29,7 +29,7 @@ manage users, teams, and repositories in the organization. [Learn more](orgs.md)
 
 ### How many organizations can I create?
 
-You can create multiple organizations. However, if you’re enabling SSO and have multiple organizations, each organization must have a domain associated with it.
+First, you should set up your first organization and contact the Customer Success team at customer-success@docker.com to add the additional organizations. Also, if you are enabling SSO, it is configured based on your domain, not your organization.
```

Comment:
First, you should set up your first organization --> `To begin, you should set up your first organization`
(removing one "first")

Reply:
@jerae-duffin Do users without a business subscription also need to contact the customer success team to create additional orgs?

Reply:
this email will only go to inside sales customers, there are only a few team subscriptions in that population, and probably would need additional hands on support from customer success managers

Reply:
Thanks @izzychu  Updated